### PR TITLE
language not case sensitive for getCode

### PR DESCRIPTION
**Pull request by angaz**
Added .toLowerCase to allow getCode(), passing 'english' or 'English' to return 'en'

### CODE REVIEW

**Comment by meikidd** [comment]
thx

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "build/index.js",
   "scripts": {
     "build": "babel src --out-dir build",
-    "test": "mocha --require babel-core/register"
+    "test": "mocha --require babel-core/register --harmony_array_includes"
```
Comment (meikidd) [inline package.json:8]: it cause `bad option: --harmony_array_includes` error on my development machine when node version is v6.0+

Comment (angaz) [reply package.json:8]: Yes, sorry, I forgot to undo that change. I didn't mean to push it.

Comment (meikidd) [reply package.json:8]: never mind. 
and v1.2.2 published

**Pull request merged**